app/js/views/services-list-item.jsx

```diff
@@ -41,6 +49,14 @@ export default class ServicesListItem extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    switch (this.service.type) {
+      case 'motion-sensor':
+        this.service.off('motion', this.onMotion);
```

Comment:
Can we do something like `this.service.offAll('motion')` here?
Or even add a new method to `EventDispatcher` to remove all listeners of all events? That'd make this part shorter and make sure we don't forget to unlisten to anything.